alembic/versions/2021031301_rename_worker_tag_column_to__6cee23f1e02f.py

```diff
@@ -0,0 +1,24 @@
+"""Rename worker tag column to tags
+
+Revision ID: 6cee23f1e02f
+Revises: db12798a7cf6
+Create Date: 2021-03-13 01:29:29.309976
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '6cee23f1e02f'
+down_revision = 'db12798a7cf6'
+
+
+def upgrade():
+    op.add_column('worker', sa.Column('tags', sa.Text(), nullable=True))
```

Comment:
I don't think this would work. This would essentially drop all existing tags.

Let's use `op.alter_column` instead (https://stackoverflow.com/a/46452540/1950269).

Reply:
good catch, thanks!